### PR TITLE
Refactor/omega 54 move poweroutage map to main category

### PR DESCRIPTION
**Pull request by lobunto**
OMEGA-54

## What is the reason for this change?
This refactoring mostly solves some design issues (and redirect) when we are going to implement the power outage map into the Main Category page.

TODO (but can be done after the PR has been merged):
- Update URL for private and pro menus with Alias to "strømbrudd" Main Category in Contentful

### What is the expected result?
- The power outage map should be available at `/strombruddskart`
- The `/kart` path should redirect to `/strombruddskart`
- Test url: `/strombruddskart`

[Url or screenshot if possible]

## How is it solved?
- Add redirect from /kart to "strømbrudd" Main Category
- Remove title and preamble for the power outage map component
- Minor style fixes: scrolling, dynamic height

## Any changes/updates that may affect others?
|                                     |Changed|
|-------------------------------------|-------|
| Unit tests                          |     |
| Dependencies                        |       |
| Environment variables               |       |
| Build scripts                       |       |

## How can it be tested and verified?

- [ ] The power outage map should be available at `/strombruddskart`
- [ ] You are able to integrate with the power outage map
- [ ] The power outage map should scale to full width on mobile devices

Screenshot 2020-09-01 at 08 34 47

### CODE REVIEW

**Comment by OysteinThuen** [comment]
Build #2999 successfully built and deployed.

- Prod: https://5f4df0a249ba2fc59a856e6d--elvia-prod.netlify.app
- Test: https://5f4df0a011d0c492d84649ea--elvia-test.netlify.app

File: packages/hdd-react/src/components/PowerOutageMap/PowerOutageMap.module.scss [hunk 1]
```diff
@@ -237,12 +216,10 @@
   height: 100%;
 
   @media #{$mobileOnly} {
-    height: 300px;
-    max-width: 420px;
+    height: 400px;
```
Comment (FredricW) [inline packages/hdd-react/src/components/PowerOutageMap/PowerOutageMap.module.scss:219]:
```suggestion
    height: emCalc(400px);
```

File: packages/hdd-react/src/components/PowerOutageMap/PowerOutageMap.module.scss [hunk 2]
```diff
@@ -237,12 +216,10 @@
...
   }
 
   @media #{$tablet} {
     height: 800px;
```
Comment (FredricW) [inline packages/hdd-react/src/components/PowerOutageMap/PowerOutageMap.module.scss:223]:
```suggestion
    height: emCalc(800px);
```

File: packages/web/src/templates/Article/Article.module.scss [hunk 3]
```diff
@@ -106,7 +106,7 @@
       margin-bottom: emCalc(56px);
     }
 
-    a {
+    & > p > a {
```
Comment (FredricW) [inline packages/web/src/templates/Article/Article.module.scss:109]: This means only the first level of a tags will get these styles. Is that what we want?

**Comment by OysteinThuen** [comment]
Build #3078 successfully built and deployed.

- Prod: https://5f4e36e976c3d80096b8a983--elvia-prod.netlify.app
- Test: https://5f4e36e442be2e00b3faff59--elvia-test.netlify.app

**Comment by OysteinThuen** [comment]
Build #3094 successfully built and deployed.

- Prod: https://5f4e4182f747cc0949647daa--elvia-prod.netlify.app
- Test: https://5f4e415352fcbe086be9eac3--elvia-test.netlify.app

File: packages/web/src/templates/Article/Article.module.scss (hunk 3 above)
Comment (lobunto) [reply packages/web/src/templates/Article/Article.module.scss]: Fixed it to what that was intentional. All links in paragraphs should have this style, not only the first level.

**Review by FredricW — APPROVED** [review]

**Review by FredricW — CHANGES_REQUESTED** [review]
Set width 100% and height 100vh and we are good :)

**Comment by lobunto** [comment]
> Set width 100% and height 100vh and we are good :)

Where? 🤔

**Comment by FredricW** [comment]
On the map :)

**Review by FredricW — APPROVED** [review]

**Pull request merged**